graphql_compiler/compiler/compiler_entities.py

```diff
@@ -43,9 +44,23 @@ def __repr__(self):
     # pylint: disable=protected-access
     def __eq__(self, other):
         """Return True if the CompilerEntity objects are equal, and False otherwise."""
-        return (type(self) == type(other) and
-                self._print_args == other._print_args and
-                self._print_kwargs == other._print_kwargs)
+        if not type(self) == type(other):
```

Comment:
Consider a `!=` here, the `not ... ==` looks a little strange to my eye.

Reply:
Where did you land on this?

Reply:
I think I just failed to push my local changes up, sorry about that.

Reply:
Pushed now.